UnityProject/Assets/Scripts/MapSaver/MapSaver.cs

```diff
@@ -947,61 +948,67 @@ public static string Matrix4X4ToString(Matrix4x4 matrix4X4, StringBuilder SB)
 			CompactObjectMapData compactObjectMapData,
 			Vector3? CoordinateOverride = null, bool UseInstance = false)
 		{
-			var RuntimeSpawned = Object.GetComponent<RuntimeSpawned>();
-			if (RuntimeSpawned != null) return;
+			if (Object.TryGetComponent<MapSaverIgnoreObject>(out var _))
```

Comment:
This would only be on the route of objects, So this wouldn't work, I think you should just pool it instead of adding a tag,  RecursiveSaveObject Is where it should go

Reply:
> This would only be on the route of objects, So this wouldn't work, I think you should just pool it instead of adding a tag, RecursiveSaveObject Is where it should go

I tried pooling it, it's a large headache.